src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -0,0 +1,24 @@
+import 'isomorphic-fetch';
+
+const fetchMarkup = async url =>
+  fetch(url, { mode: 'no-cors' })
+    .then(html => {
+      return html.text().then(text => {
+        return text;
+      });
+    })
+    .catch(e => console.error(`HTTP Error: "${e}"`));
```

Comment:
No big deal but I try and avoid promises where possible with `async/await` being available in javascript.

The could be:
```javascript
const fetchMarkup = async url => {
  const html = await fetch(url, { mode: 'no-cors' });
  try {
    return await html.text();
  } catch (e) {
    console.error(`HTTP Error: "${e}"`);
  }
}
```
Also, do console.errors make it through to sumologic from the server-side render? If they do, we could identify any problematic includes by looking through logs, particularly if we get reports some are not loading? Maybe change the wording here to be more specific e.g. `Error fetching include ${url}: ${e}`

Reply:
Sorry just edited that to ensure error is the fetch is captured by the exception handler